extension/js/common/inject.ts

```diff
@@ -62,12 +62,39 @@ export class Injector {
       .append(this.factory.metaStylesheet('webmail') + this.factory.metaNotificationContainer()); // xss-safe-factory
   };
 
-  public openComposeWin = (draftId?: string): boolean => {
+  public openComposeWin = (
+    draftId?: string,
+    fullscreen = false,
+    composeMsgDetails?: { subject?: string; recipients?: { to?: string[]; cc?: string[]; bcc?: string[] } } | undefined
```

Comment:
let's create a separate type for it, you can check https://github.com/FlowCrypt/flowcrypt-browser/blob/master/extension/chrome/elements/compose-modules/compose-types.ts, there are similar types there

Reply:
Thanks for the hint - I have created a type definition for that in the appropriate place it should be. Thank you!